controllers/tasks.js

```diff
@@ -0,0 +1,98 @@
+const Task = require('../models/Task')
+
+const addTask = (req, res) => {
+    res.render('pages/addTask')
+}
+
+const createTask = async (req, res) => {
+    try {
+        if (req.body.complete) {
+            req.body.completed=true
+        }
+        console.log(req.body, "Inside the create func")
```

Comment:
Remember to remove console logs for debugging before committing your code. Better yet, get in the habit of using a debugger instead so you can inspect your code's behavior without needing to change the code at all! If you're interested in learning more, please just ask; I can link you to resources.

Reply:
I dont know how to use debugger for the backend code?  when I am in the ev tools I am not able to find the controllers file or any of the files in the source so i used the console.logs to check my code. With React I would be debugging my code. 

If you can share the link would be nice. Thanks